docs/guide/deployment/authorization-server.md

```diff
@@ -19,47 +19,82 @@ All requests to service must contains valid access token in [grpc metadata](http
-| `-` | `LISTEN_ACME_DIRECTORY_URL` | string |  `url of acme directory` | `""` |
-| `-` | `LISTEN_ACME_DOMAINS` | string | `list of domains for which will be in certificate provided from acme` | `""` |
-| `-` | `LISTEN_ACME_REGISTRATION_EMAIL` | string | `registration email for acme` | `""` |
-| `-` | `LISTEN_ACME_TICK_FREQUENCY` | string | `interval of validate certificate` | `""` |
-| `-` | `LISTEN_ACME_USE_SYSTEM_CERTIFICATION_POOL` | bool | `load CAs from system` | `false` |
-| `-` | `LISTEN_FILE_CA_POOL` | string | `path to pem file of CAs` |  `""` |
-| `-` | `LISTEN_FILE_CERT_KEY_NAME` | string | `name of pem certificate key file` | `""` |
-| `-` | `LISTEN_FILE_CERT_DIR_PATH` | string | `path to directory which contains LISTEN_FILE_CERT_KEY_NAME and LISTEN_FILE_CERT_NAME` | `""` |
-| `-` | `LISTEN_FILE_CERT_NAME` | string | `name of pem certificate file` | `""` |
-| `-` | `LISTEN_FILE_USE_SYSTEM_CERTIFICATION_POOL` | bool | `load CAs from system` | `false` |
-| `-` | `LOG_ENABLE_DEBUG` | bool | `enable debugging message` | `false` |
-| `-` | `MONGODB_URI` | string | `uri to mongo database` | `"mongodb://localhost:27017"` |
-| `-` | `MONGODB_DATABASE` | string | `name of database` | `"authorization"` |
-| `-` | `LOG_ENABLE_DEBUG` | bool | `debug logging` | `false` |
+| Key | Type | Description | Default |
+| ---------- | -------- | -------------- | ------- |
+| `log.debug` | bool | `enable debugging message` | `false` |
+| `api.grpc.address` | string | `listen address` | `"0.0.0.0:9100"` |
+| `api.grpc.tls.caPool` | string | `file path to the root certificates in PEM format` |  `""` |
+| `api.grpc.tls.keyFile` | string | `file name of private key in PEM format` | `""` |
+| `api.grpc.tls.certFile` | string | `file name of certificate in PEM format` | `""` |
+| `api.grpc.tls.clientCertificateRequired` | bool | `require client certificate` | `true` |
+| `api.grpc.authorization.authority` | string | `endpoint of oauth provider` | `""` |
+| `api.grpc.authorization.audience` | string | `audience of oauth provider` | `""` |
+| `api.grpc.authorization.ownerClaim` | string | `owner claim of oauth provider` | `"sub"` |
+| `api.grpc.authorization.http.maxIdleConns` | int | `controls the maximum number of idle (keep-alive) connections across all hosts. Zero means no limit.` | `16` |
+| `api.grpc.authorization.http.maxConnsPerHost` | int | `optionally limits the total number of connections per host, including connections in the dialing, active, and idle states. On limit violation, dials will block. Zero means no limit.` | `32` |
+| `api.grpc.authorization.http.maxIdleConnsPerHost` | int | `if non-zero, controls the maximum idle (keep-alive) connections to keep per-host. If zero, DefaultMaxIdleConnsPerHost is used.` | `16` |
+| `api.grpc.authorization.http.idleConnTimeout` | int | `the maximum amount of time an idle (keep-alive) connection will remain idle before closing itself. Zero means no limit.` | `30` |
```

Comment:
for duration leafs:

- idleConnTimeout
- timeout

they are string type and format is defined by https://golang.org/pkg/time/#example_ParseDuration

Reply:
Pls add there mentioned something like: `The string is decimal numbers, each with optional fraction and a unit suffix, such as "300ms", "1.5h" or "2h45m". Valid time units are "ns", "us", "ms", "s", "m", "h". `

Reply:
I added your comment at bottom of each md files. See latest commit (https://github.com/plgd-dev/cloud/pull/248/commits/c7a184c03bb4a63bbba9cde90948775834ae06aa).